toolkit/scss/forms/_checkbox-tree.scss

```diff
@@ -5,7 +5,7 @@
 
 @include fake-full-width('.checkbox-tree');
 
-fieldset.checkbox-tree__selection-buttons > legend {
+fieldset.checkbox-tree__inputs > legend {
   @extend %visuallyhidden;
```

Comment:
Yeah, we do this (almost) everywhere we use the `visuallyhidden` stuff.  Totally could be a mixin.